### PR TITLE
Close WhatsNewActivity if open

### PR DESCRIPTION
**Pull request by Unpublished**
Follow up of #4044 

See #4048

### CODE REVIEW

**Comment by Unpublished** [comment]
Good to wait for CI 😅 Added an import for `WhatsNewActivity`.

File: src/androidTest/java/com/owncloud/android/ui/activity/FileDisplayActivityTest.java [hunk 1]
```diff
@@ -17,6 +23,12 @@
     @Test
     public void testSetupToolbar() {
         try (ActivityScenario<FileDisplayActivity> scenario = ActivityScenario.launch(FileDisplayActivity.class)) {
+            InstrumentationRegistry.getInstrumentation().runOnMainSync(() -> {
+                Activity activity = ActivityLifecycleMonitorRegistry.getInstance().getActivitiesInStage(RESUMED).iterator().next();
```
Comment (tobiasKaminsky) [inline src/androidTest/java/com/owncloud/android/ui/activity/FileDisplayActivityTest.java]: linewrap at 120, please

**Comment by tobiasKaminsky** [comment]
Just a small thing, otherwise good to go :)

File: src/androidTest/java/com/owncloud/android/ui/activity/FileDisplayActivityTest.java (hunk 1 above)
Comment (Unpublished) [reply src/androidTest/java/com/owncloud/android/ui/activity/FileDisplayActivityTest.java]: Done

**Comment by Unpublished** [comment]
@tobiasKaminsky Added https://github.com/Unpublished/android/commit/1ea47a00db9e9cc6ced1cd6f3072f98bfcf77315 locally, to ensure linewrapping in future.
Should I add it to this repo?

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky Added [Unpublished@1ea47a0](https://github.com/Unpublished/android/commit/1ea47a00db9e9cc6ced1cd6f3072f98bfcf77315) locally, to ensure linewrapping in future.
> Should I add it to this repo?

Would be good, but then please in a new PR :-)

**Pull request merged**